packages/mui-styled-engine/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine",
-  "version": "5.0.1",
+  "version": "5.0.2",
```

If we were not using `"version": "independent"` in `/lerna.json`, I believe that Lerna would have used

```suggestion
  "version": "5.0.6",
```

here. semver allows to skip versions, and I would argue is clearer for developers.